### ISSUE TITLE
Compilation failure

### ISSUE THREAD

**Issue by salamani** [jnr/jffi#82]
Using java -version
openjdk version "11.0.7" 2020-04-14
OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.7+10)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.7+10, mixed mode)

Ant compilation failed with error
```
...
-generate-native-headers:
    [mkdir] Skipping /home/test/jffi-jffi-1.2.23/build/jni because it already exists.
    [mkdir] Skipping /home/test/jffi-jffi-1.2.23/build/classes because it already exists.
    [javah] Compilation arguments:
    [javah] '-d'
    [javah] '/home/test/jffi-jffi-1.2.23/build/jni'
    [javah] '-classpath'
    [javah] '/home/test/jffi-jffi-1.2.23/build/classes'
    [javah] '-force'
    [javah]
    [javah] The ' characters around the executable and arguments are
    [javah] not part of the command.
    [javah] Classes to be compiled:
    [javah]     com.kenai.jffi.Foreign
    [javah]     com.kenai.jffi.ObjectBuffer
    [javah]     com.kenai.jffi.Version

BUILD FAILED
/home/test/jffi-jffi-1.2.23/build.xml:300: Can't load javah
        at org.apache.tools.ant.taskdefs.optional.javah.SunJavah.compile(SunJavah.java:60)
        at org.apache.tools.ant.taskdefs.optional.Javah.execute(Javah.java:472)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:292)
        at jdk.internal.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:435)
        at org.apache.tools.ant.Target.performTasks(Target.java:456)
        at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1393)
        at org.apache.tools.ant.Project.executeTarget(Project.java:1364)
        at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
        at org.apache.tools.ant.Project.executeTargets(Project.java:1248)
        at org.apache.tools.ant.Main.runBuild(Main.java:851)
        at org.apache.tools.ant.Main.startAnt(Main.java:235)
        at org.apache.tools.ant.launch.Launcher.run(Launcher.java:280)
        at org.apache.tools.ant.launch.Launcher.main(Launcher.java:109)
Caused by: java.lang.ClassNotFoundException: com.sun.tools.javah.Main
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:315)
        at org.apache.tools.ant.taskdefs.optional.javah.SunJavah.compile(SunJavah.java:57)
        ... 17 more

Total time: 0 seconds
```

**Comment by huiji12321** [comment]
javah does not exist under Java 10 and higher,I tried "apt-get install openjdk-8-jdk",it works!

**Comment by headius** [comment]
Ah yes, we may need some patching for the build to use whatever replaced `javah`. Help?

**Comment by TobiX** [comment]
Shouldn't this been fixed since #84?

**Comment by headius** [comment]
@TobiX Right you are!
